includes/Core/REST_API/REST_Routes.php

```diff
@@ -79,6 +88,7 @@ public function __construct( Context $context, Authentication $authentication =
 			$modules = new Modules( $this->context, null, null, $this->authentication );
 		}
 		$this->modules = $modules;
+		$this->input   = new Input();
```

Comment:
We access this everywhere via the instance in `Context` (available via `$this->context->input`. This is so we can use `MutableInput` in tests and there is only one place to swap this out because the same context instance is shared in internal dependencies. Using a local instance is unnecessary and would make this more difficult. Please use `$this->context->input` instead 😄

Reply:
Updated :)